src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,771 @@
+service/nginx exposed
+```
+
+Verify the Service exists and is running:
+
+```bash
+$ kubectl -n nginx-test get service nginx
+NAME    TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+nginx   ClusterIP   100.2.29.136   <none>        8080/TCP   1m40s
+```
+
+```bash
+$ kubectl -n nginx-test get pods -l app=nginx -o wide
+NAME                     READY   STATUS    RESTARTS   AGE   IP           NODE              NOMINATED NODE   READINESS GATES
+nginx-5578584966-d7sj7   1/1     Running   0          22s   10.2.224.3   cluster2-worker   <none>           <none>
+```
+
+##### 5. Consume the Service from **cluster3**
+
+```bash
+$ kubectl config use-context cluster3
+Switched to context "cluster3".
+```
+
+Run a test Pod on **cluster3** and try to access the `nginx` Service from within the Pod:
+
+```bash
+$ kubectl -n nginx-test  run --generator=run-pod/v1 \
+tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+```
+
+```bash
+bash-5.0# curl nginx.nginx-test.svc.clusterset.local:8080
+<!DOCTYPE html>
+<html>
+<head>
+<title>Welcome to nginx!</title>
+<style>
+    body {
+        width: 35em;
+        margin: 0 auto;
+        font-family: Tahoma, Verdana, Arial, sans-serif;
+    }
+</style>
+</head>
+<body>
+<h1>Welcome to nginx!</h1>
+<p>If you see this page, the nginx web server is successfully installed and
+working. Further configuration is required.</p>
+
+<p>For online documentation and support please refer to
+<a href="http://nginx.org/">nginx.org</a>.<br/>
+Commercial support is available at
+<a href="http://nginx.com/">nginx.com</a>.</p>
+
+<p><em>Thank you for using nginx.</em></p>
+</body>
+</html>
+```
+
+<!-- markdownlint-disable no-hard-tabs -->
+```bash
+bash-5.0# dig nginx.nginx-test.svc.clusterset.local
+; <<>> DiG 9.16.6 <<>> nginx.nginx-test.svc.clusterset.local
+;; global options: +cmd
+;; Got answer:
+;; WARNING: .local is reserved for Multicast DNS
+;; You are currently testing what happens when an mDNS query is leaked to DNS
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 55022
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 6f9db9800a9a9779 (echoed)
+;; QUESTION SECTION:
+;nginx.nginx-test.svc.clusterset.local. IN	A
+
+;; ANSWER SECTION:
+nginx.nginx-test.svc.clusterset.local. 5 IN A	100.3.220.176
+
+;; Query time: 5 msec
+;; SERVER: 100.3.0.10#53(100.3.0.10)
+;; WHEN: Tue Dec 01 07:45:48 UTC 2020
+;; MSG SIZE  rcvd: 125
+```
+<!-- markdownlint-enable no-hard-tabs -->
+
+{{% notice note %}}
+At this point we have the same `nginx` Service deployed within the nginx-test namespace on both clusters. Note that DNS resolution works,
+and the IP address **100.3.220.176** returned is the ClusterIP associated with the *local* nginx Service deployed on **cluster3**. This is
+expected, as Submariner prefers to handle the traffic locally whenever possible.
```

Comment:
This doesn't actually test that the service from **cluster2** is available since `dig` is run from **cluster3** which will always return the local `ClusterIP`. We should run `dig` from `cluster2` and verify its local `ClusterIP` is returned.